proposed/storageos-operator/v0.2.12/templates/psp.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.podSecurityPolicy.enabled }}
+
+apiVersion: extensions/v1beta1
```

Comment:
can we use `policy/v1beta1` instead for k8s v1.16 support.

Reply:
sure.